DEBUG 2020-07-06 01:23:28.131 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-07-06 01:23:28.131 UTC: 	 Running on: Linux lxplus775.cern.ch 3.10.0-1127.8.2.el7.x86_64 #1 SMP Tue May 12 16:57:42 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-07-06 01:23:28.145 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-07-06 01:23:28.145 UTC: 	 Executing command: 'status'
DEBUG 2020-07-06 01:23:28.145 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jipeng/.crab3
ERROR 2020-07-06 01:23:28.146 UTC: 	 Error: crab_full_run2_2018_WG1Jets_version1 is not a valid CRAB project directory.
ERROR 2020-07-06 01:23:28.146 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: crab_full_run2_2018_WG1Jets_version1 is not a valid CRAB project directory.
INFO 2020-07-06 01:23:28.149 UTC: 	 Log file is /afs/cern.ch/work/j/jipeng/fullRun2VBSWA/18/tight/CMSSW_10_2_18/src/VAJets/PKUTreeMaker/test/crab.log
DEBUG 2020-07-06 01:23:28.434 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-07-06 01:23:28.434 UTC: 	 Running on: Linux lxplus775.cern.ch 3.10.0-1127.8.2.el7.x86_64 #1 SMP Tue May 12 16:57:42 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-07-06 01:23:28.447 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-07-06 01:23:28.447 UTC: 	 Executing command: 'status'
DEBUG 2020-07-06 01:23:28.447 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jipeng/.crab3
ERROR 2020-07-06 01:23:28.448 UTC: 	 Error: crab_full_run2_2018_WG2Jets_version1 is not a valid CRAB project directory.
ERROR 2020-07-06 01:23:28.448 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: crab_full_run2_2018_WG2Jets_version1 is not a valid CRAB project directory.
INFO 2020-07-06 01:23:28.450 UTC: 	 Log file is /afs/cern.ch/work/j/jipeng/fullRun2VBSWA/18/tight/CMSSW_10_2_18/src/VAJets/PKUTreeMaker/test/crab.log
DEBUG 2020-07-06 13:28:26.850 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-07-06 13:28:26.851 UTC: 	 Running on: Linux lxplus701.cern.ch 3.10.0-1127.8.2.el7.x86_64 #1 SMP Tue May 12 16:57:42 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-07-06 13:28:26.864 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-07-06 13:28:26.864 UTC: 	 Executing command: 'status'
DEBUG 2020-07-06 13:28:26.864 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jipeng/.crab3
ERROR 2020-07-06 13:28:26.865 UTC: 	 Error: crab_full_run2_2018_WG1Jets_version1 is not a valid CRAB project directory.
ERROR 2020-07-06 13:28:26.865 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: crab_full_run2_2018_WG1Jets_version1 is not a valid CRAB project directory.
INFO 2020-07-06 13:28:26.869 UTC: 	 Log file is /afs/cern.ch/work/j/jipeng/fullRun2VBSWA/18/tight/CMSSW_10_2_18/src/VAJets/PKUTreeMaker/test/crab.log
DEBUG 2020-07-06 13:28:27.220 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-07-06 13:28:27.221 UTC: 	 Running on: Linux lxplus701.cern.ch 3.10.0-1127.8.2.el7.x86_64 #1 SMP Tue May 12 16:57:42 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-07-06 13:28:27.237 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-07-06 13:28:27.237 UTC: 	 Executing command: 'status'
DEBUG 2020-07-06 13:28:27.237 UTC: 	 Found CRAB cache file /afs/cern.ch/user/j/jipeng/.crab3
ERROR 2020-07-06 13:28:27.238 UTC: 	 Error: crab_full_run2_2018_WG2Jets_version1 is not a valid CRAB project directory.
ERROR 2020-07-06 13:28:27.238 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: crab_full_run2_2018_WG2Jets_version1 is not a valid CRAB project directory.
INFO 2020-07-06 13:28:27.240 UTC: 	 Log file is /afs/cern.ch/work/j/jipeng/fullRun2VBSWA/18/tight/CMSSW_10_2_18/src/VAJets/PKUTreeMaker/test/crab.log
